app/Domain/Tickets/Templates/showAll.blade.php

```diff
@@ -126,261 +126,238 @@
                                 #<?= $tpl->e($row['id']) ?>
                             </td>
 
-                        <td data-order="<?=$tpl->e($row['headline']); ?>">
-                            <?php if ($row['dependingTicketId'] > 0) { ?>
-                                <small><a href="#/tickets/showTicket/<?=$row['dependingTicketId'] ?>"><?=$tpl->escape($row['parentHeadline']) ?></a></small> //<br />
-                            <?php } ?>
-                            <a href="#/tickets/showTicket/<?=$tpl->e($row['id']); ?>"><?=$tpl->e($row['headline']); ?></a></td>
+                            <td data-order="<?= $tpl->e($row['headline']) ?>">
+                                <?php if ($row['dependingTicketId'] > 0) { ?>
+                                <small><a
```

Comment:
Is this the code formatting tool again? Why was that changed?

Reply:
most likely